src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/AddAStandard/SelectAStandardController.cs

```diff
@@ -55,15 +55,14 @@ public async Task<IActionResult> SubmitAStandard(SelectAStandardSubmitModel subm
 
             var standardInformation = await _mediator.Send(new GetStandardInformationQuery(submitModel.SelectedLarsCode));
 
-            if (string.IsNullOrEmpty(standardInformation.RegulatorName))
+            if (!string.IsNullOrEmpty(standardInformation.RegulatorName) && standardInformation.Regulated && standardInformation.IsRegulatedForProvider)
```

Comment:
We only need to check IsRegulatedForProvider now

Reply:
Perhaps remove Regulated if it is a calculated property, seems redundant

Reply:
As discussed it is better if we restrict the use of `Regulated` logic with in the web api. So here we need not interpret it at all.

Reply:
Removed as requested